app/router.js

```diff
@@ -8,7 +8,7 @@ const Router = EmberRouter.extend({
 
 Router.map(function() {
   this.route('form', { path: 'form/:id' });
-  this.route('search');
+  this.route('searches', { path: 'searches' });
```

Comment:
you don't need to declare a path if the path is the same as the path name

Reply:
corrected